pym/bob/languages.py

```diff
@@ -599,9 +567,150 @@ def setupFingerprint(spec, env):
         return [interpreter, "-c", spec.fingerprintScript]
 
 
+class PythonResolver(IncludeResolver):
+    def _includeFile(self, name):
+        return 'os.path.join(_BOB_TMP_BASE, ' + repr(name) + ')'
+
+    def _includeLiteral(self, content):
+        return repr(content.decode('utf8'))
+
+
+class PythonLanguage:
+    index = ScriptLanguage.PYTHON
+    glue = "\nos.chdir(os.environ['BOB_CWD'])\n"
+    Resolver = PythonResolver
+
+    HELPERS = dedent("""\
+        from subprocess import run, call, check_call, check_output
```

Comment:
Not sure if I'm fully understanding what's going on here, but it looks like these are inserted into the final script. If this is the case: do you really want to clutter every user-provided script with those functions? Especially `run` and `call` are pretty generic and might be used by the user. I'd prefer `import subprocess` like the ones below, even if that means more work here.

Reply:
Yeah, that's something where I'm not yet sure about. At least some minimal imports are required by the script prologue (os, os.path, sys). Almost any script *will* use the subprocess main functions anyway. I first found it convenient to implicitly import them always. But thinking more about this it might not be a good idea and leave this entirely to the user. That can be handled by some class(es) and `{checkout,build,package}SetupPython` snippets.